packages/page-staking/src/Performance/CurrentList.tsx

```diff
@@ -158,36 +61,61 @@ function CurrentList ({ className, currentSessionCommittee, favorites, session,
     ]
   );
 
+  function calculatePercentReward (blocksCreated: number, blocksTargetValue: number) {
+    let rewardPercentage = 0;
+
+    if (blocksTargetValue > 0) {
+      rewardPercentage = 100 * blocksCreated / blocksTargetValue;
+    } else if (blocksTargetValue === 0 && blocksCreated === 0) {
+      rewardPercentage = 100;
```

Comment:
in this case, the value can be returned right inside this `if`, because the next if will always be true for the value of 100, so there is no need to check it

Reply:
Good point, this one `if` belongs to another `if` condition